### DISCUSSION TITLE
Unable to stream to simple RTSP server on Azure ubuntu VM

### DISCUSSION THREAD

**Discussion by yvkrishna91** [bluenviron/mediamtx#668] [Questions]
Server: docker run --rm -it --network=host aler9/rtsp-simple-server
2021/10/26 09:20:38 I [0/0] rtsp-simple-server v0.17.3 
2021/10/26 09:20:38 I [0/0] [RTSP] UDP/RTP listener opened on :8000 
2021/10/26 09:20:38 I [0/0] [RTSP] UDP/RTCP listener opened on :8001 
2021/10/26 09:20:38 I [0/0] [RTSP] TCP listener opened on :8554 
2021/10/26 09:20:38 I [0/0] [RTMP] listener opened on :1935 
2021/10/26 09:20:38 I [0/0] [HLS] listener opened on :8888 
2021/10/26 09:22:27 I [0/0] [RTSP] [conn 49.207.214.142:32115] opened 
2021/10/26 09:22:28 I [0/0] [RTSP] [session 299851061] opened by 49.207.214.142:32115 
2021/10/26 09:22:28 I [0/0] [path 002] created 
2021/10/26 09:22:29 I [1/0] [RTSP] [session 299851061] is publishing to path '002', 1 track with UDP 
2021/10/26 09:22:38 I [0/0] [path 002] destroyed 
2021/10/26 09:22:38 I [0/0] [RTSP] [session 299851061] closed

Stream gets destroyed after it starts publishing..
Client streams from an IP camera using gstreamer:
gst-launch-1.0 rtspsrc location=rtsp://192.168.0.101:554/Streaming/channels/1 latency=200 buffer-mode=auto short-header=TRUE ! rtph264depay ! video/x-h264, format=avc,alignment=au ! rtspclientsink location=rtsp://20.198.243.60:8554/002

Inbound rules are set for - 8000(tcp/udp), 8001(udp),8554(tcp)

Need help in debugging the issue.  
I could get the same working when rtsp server is hosted on localhost.

**Comment by aler9** [comment]
Hello, please try again with v0.17.7, that prints the exact reason for a session being destroyed. In this way you can debug the situation more clearly.

[a comment by an automated account is not shown]
